src/components/Sections/SectionDefaultWave.svelte

```diff
@@ -0,0 +1,3 @@
+<div
+  class={`lg:sm-[url(/src/lib/assets/placeholder-portal.svg)] absolute left-0 h-full bg-left-bottom bg-no-repeat sm:-bottom-16 sm:max-h-[100px] sm:w-[10%] sm:bg-[url(/src/lib/assets/section3mobile.svg)] md:max-h-full md:w-full lg:bottom-0 lg:max-w-full`}
+></div>
```

Comment:
what should i change here to fit the image in on the side?

![image](https://github.com/user-attachments/assets/8cde934b-76e3-425e-b7c1-574ae9f44a98)